src/components/molecules/Message/Message.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box } from "@mui/material";
+import { MessageText, UserAvatar } from "components";
+import styles from "./Message.module.scss";
+
+type MessageProps = {
+  author: {
+    name: string;
+    avatar?: string;
+  };
+  message: {
+    content: string;
+    date: Date;
+  };
+  isOwn?: boolean;
+  isLast?: boolean;
+};
+
+export const Message = ({ author, message, isOwn, isLast }: MessageProps) => {
+  return (
+    <>
+      <Box
+        className={`${isOwn ? styles.messageToRight : styles.messageToLeft} ${
+          styles.messageContainer
+        }`}
```

Comment:
Pomysłowo, ale tutaj musimy użyć czegoś innego.
Zerknij tutaj, są przykłady. W Css modules powinniśmy użyć czegoś takiego.

To bardzo popularna biblioteka, wykorzystywana w wielu firmach.
Idealna do Css modules
https://www.npmjs.com/package/classnames